sensitivity-analysis/src/main/java/com/farao_community/farao/sensitivity_analysis/SystematicSensitivityResult.java

```diff
@@ -166,51 +200,41 @@ public void setStatus(SensitivityComputationStatus status) {
         this.status = status;
     }
 
-    public double getReferenceFlow(FlowCnec cnec) {
+    public double getReferenceFlow(FlowCnec cnec, Side side) {
         StateResult stateResult = getCnecStateResult(cnec);
-        if (stateResult == null) {
+        if (stateResult == null ||
+            !stateResult.getReferenceFlows().containsKey(cnec.getNetworkElement().getId())) {
```

Comment:
it might make more sense to put all three cases where we return 0 here 
(stateResult == null || 
!stateResult.getReferenceFlows().containsKey(cnec.getNetworkElement().getId()) || 
!stateResult.getReferenceFlows().get(cnec.getNetworkElement().getId()).containsKey(side))